### PR TITLE
EC-220: User management revision

### PR DESCRIPTION
**Pull request by DominikaK**
| Question      | Answer
| ------------- | ---
| JIRA Ticket   | https://issues.ibexa.co/browse/EC-220
| Versions      | 3.1, 3.2, master

### CODE REVIEW

File: docs/guide/user_management/token.md [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+    $validUntil,
+    $actionService,
+    $actionServiceMethod,
+    Token::TOKEN_TYPE_ONE_TIME
+);
+```
+
+## Using the token
+
+You can use the token in the built-in token controller to call a service and method, or in a custom controller or service.
```
Comment (dabrt) [inline docs/guide/user_management/token.md:41]:
```suggestion
You can use the token to call a service and method in the built-in token controller, or in a custom controller or service.
```
Is it 
"... in the built-in token controller that calls a service and method..."
or
"the token to call a service and method in the built-in token controller, or in a custom controller or service"
?

File: docs/guide/user_management/token.md [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+
+### Checking the token
+
+The token can be checked using `TokenService`. It returns the parameters stored when the token was created:
```
Comment (dabrt) [inline docs/guide/user_management/token.md:45]:
```suggestion
The token can be checked with the `TokenService`. It returns the parameters that were stored when the token was created: 
```

File: docs/guide/user_management/token.md [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+/** @var Token $token */
+$token = $this->tokenService->loadToken($userToken);
+
+$serviceMethod = $token->getActionServiceMethod();
+$tokenParameter = $token->getActionServiceMethodParameter();
+```
+
+#### TokenController
+
+You can use the token to distribute unique links to customers.
```
Comment (dabrt) [inline docs/guide/user_management/token.md:58]:
```suggestion
You can use the token to deliver unique links to customers.
```

File: docs/guide/user_management/token.md [hunk 4]
```diff
@@ -0,0 +1,97 @@
...
+
+`TokenController` is available at the `/token/[token]` route, for example: `/token/124f564f6d4df4fd3fd4df34fd34fd`.
+
+`TokenController` is called to process a token, e.g. after user registration when the user clicks the activation link.
```
Comment (dabrt) [inline docs/guide/user_management/token.md:62]:
```suggestion
`TokenController` is called to process a token, for example, after user registration when the user clicks the activation link.
```

File: docs/guide/user_management/token.md [hunk 5]
```diff
@@ -0,0 +1,97 @@
...
+This controller loads the token from the database, activates the user account and invalidates the token.
```
Comment (dabrt) [inline docs/guide/user_management/token.md:63]:
```suggestion
The controller loads the token from the database, activates the user account and invalidates the token.
```

File: docs/guide/user_management/token.md [hunk 6]
```diff
@@ -0,0 +1,97 @@
...
+
+Each token has the following attributes to process the token-specific logic:
```
Comment (dabrt) [inline docs/guide/user_management/token.md:65]:
```suggestion
Each token has the following attributes that are processed by the token-specific logic:
```

**Review by dabrt — CHANGES_REQUESTED** [review]

File: docs/guide/newsletter/newsletter.md [hunk 7]
```diff
@@ -61,15 +68,31 @@ but you have to implement a custom event listener for that.
 
 Attributes that do not exist in the newsletter provider have to be created first.
 
-## Double-opt-in process
+#### Additional newsletter data
+
+To send additional data to the newsletter provider, you need to implement an event listener
```
Comment (DominikaK) [inline docs/guide/newsletter/newsletter.md:73]:
```suggestion
To send additional data to the newsletter provider, implement an event listener
```

File: docs/guide/user_management/login_and_registration.md [hunk 8]
```diff
@@ -0,0 +1,94 @@
...
+```
+
+The email message contains the name of the user and a link to the user in the Back Office.
+To generate a correct link to the Back Office, configure `related_admin_site_access`:
+
+``` yaml
+siso_core.default.related_admin_site_access: 'admin'
+```
+
+To adapt the success message of the private registration, change the text module with the identifier `success_register_private`.
```
Comment (DominikaK) [inline docs/guide/user_management/login_and_registration.md:38]:
```suggestion
To adapt the success message of the private registration, modify the text module with the identifier `success_register_private`.
```

File: docs/guide/user_management/token.md (hunk 1 above)
Comment (DominikaK) [inline docs/guide/user_management/token.md:41]:
```suggestion
You can use the token in the built-in token controller when calling a service and method, or in a custom controller or service.
```

**Review by dabrt — APPROVED** [review]

**Pull request merged**